hosted/gke/p1/p1_provisioning_test.go

```diff
@@ -130,6 +130,52 @@ var _ = Describe("P1Provisioning", func() {
 		Expect(err.Error()).To(ContainSubstring("not found"))
 	})
 
+	It("should be able to create a cluster with CP K8s version v-XX-a and NP K8s version v-XX should use v-XX-1 for both CP and NP", func() {
```

Comment:
```suggestion
	It("should be able to create a cluster with CP K8s version v-XX-1 and NP K8s version v-XX should use v-XX-1 for both CP and NP", func() {
```
to match with QASE title